src/rpc/net.cpp

```diff
@@ -941,6 +942,7 @@ static RPCHelpMan addpeeraddress()
 
     const std::string& addr_string{request.params[0].get_str()};
     const uint16_t port{static_cast<uint16_t>(request.params[1].get_int())};
+    const bool tried{request.params[2].isTrue()};
```

Comment:
Nit: there's nothing wrong with declaring and setting a local `tried` variable at the top of the function, but standard best practice would be to declare the variable where it's needed (https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rnr-top), or better yet, don't even use a local variable and just test `request.params[2].isTrue()` directly below.

Reply:
That's true. It's subjective, but giving each of the params a name together in one place seems to make the code easier to understand in this case.

Reply:
Sure. It's a style nit. Feel free to ignore.